packages/core/parcel/src/cli.js

```diff
@@ -151,6 +151,7 @@ program
   .command('help [command]')
   .description('display help information for a command')
   .action(function(command) {
+    // $FlowFixMe[incompatible-use]
     let cmd = program.commands.find(c => c.name() === command) || program;
```

Comment:
This is probably caused because the flow typings for commander 2 overwrite the ones for version 7. Could the new CLI also use version 7 instead?